### PR TITLE
Mailer in production

### PR DESCRIPTION
**Pull request by ejohnst2**

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ def create
     if trip.contribution.amount > 0
       contribution_order = Order.create!(sku: trip.contribution.fund.title, amount: trip.contribution.amount, state: 'pending', trip_id: trip.id, contribution: true )
     end
+    # UserMailer.trip_request(trip, trip.email).deliver_now
```
Comment (nicksarafa) [inline app/controllers/orders_controller.rb]: 💣

File: app/mailers/test_mailer.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+# class TestMailer < ActionMailer::Base
+#   def message
+#     mail(
+#       :subject => 'Hello from Postmark',
+#       :to  => 'eli@millinnolab.org',
+#       :from => 'eli@troveko.com',
+#       :html_body => '<strong>Hello</strong> dear Postmark user.',
+#       :track_opens => 'true')
+#   end
+# end
```
Comment (nicksarafa) [inline app/mailers/test_mailer.rb]: tests are usually good, mmk

File: app/models/order.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 class Order < ApplicationRecord
+  # after_create :send_request_email
```
Comment (nicksarafa) [inline app/models/order.rb]: 💣

File: app/views/user_mailer/request.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<p><%= @greeting %></p>
+
+<p>Your request to stay at <%= @order.trip.experience.title %> has been received and you can expect confirmation within 24 hours.</p>
+
+<p>Thanks for joining us on our mission to change travel.</p>
+
+<p>Speak Soon,</p>
```
Comment (nicksarafa) [inline app/views/user_mailer/request.html.erb]: `With Love,`

File: app/views/user_mailer/trip_confirmation.html.erb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<p><%= @greeting %></p>
+
+<p>Your request to stay at <%= @trip.experience.title %> has been confirmed by the host, so pack your bags!</p>
+
+<p>Thanks for joining us on our mission to change travel.</p>
+
+<p>Speak Soon,</p>
```
Comment (nicksarafa) [inline app/views/user_mailer/trip_confirmation.html.erb]: `Much Love,`

File: app/views/user_mailer/welcome.text.erb [hunk 6]
```diff
@@ -1,3 +1,9 @@
 User#welcome
 
-<%= @greeting %>, find me in app/views/user_mailer/welcome.text.erb
+<%= @greeting %>, welcome to Troveko. We are on a mission to inspire sustainable, community-based tourism initiatives in Bali and beyond, so glad to have you be a part of it.
+
+Find me in app/views/user_mailer/welcome.text.erb
+
+Best,
```
Comment (nicksarafa) [inline app/views/user_mailer/welcome.text.erb]: `Love,`

File: config/environments/production.rb [hunk 7]
```diff
@@ -17,9 +17,9 @@
   config.action_controller.perform_caching = true
 
   #actionmailer user PostMark API
-  config.action_mailer.delivery_method     = :postmark
-  config.action_mailer.postmark_settings   = { api_key: ENV['POSTMARK_API_KEY'] }
-  config.action_mailer.default_url_options = { host: "troveko.org" }
+  # config.action_mailer.delivery_method     = :postmark
+  # config.action_mailer.postmark_settings   = { api_key: ENV['POSTMARK_API_KEY'] }
+  # config.action_mailer.default_url_options = { host: "troveko.org" }
```
Comment (nicksarafa) [inline config/environments/production.rb]: 💣

File: app/views/user_mailer/welcome.text.erb (hunk 6 above)
Comment (ejohnst2) [reply app/views/user_mailer/welcome.text.erb]: lmao

**Pull request merged**